src/app/notifications-page/notifications-list/notifications-list.component.html

```diff
@@ -5,10 +5,10 @@
   Notifications
 </div>
 <div class="global__top-bar__height"></div>
```

Comment:
updated naming of `loadingFirstPage` and `loadingNextPage` to match those in `CreatorProfilePostsComponent `